contracts/teller-network/EscrowRelay.sol

```diff
@@ -164,6 +164,10 @@ contract EscrowRelay is RelayRecipient, Ownable {
       fSign := mload(add(encoded_function, add(0x20, 0)))
     }
 
+    if (relay == address(0) || transaction_fee == 0) {
```

Comment:
Can we ignore this warning?... just to avoid spending gas... probably it's not even much, but even if this function is `view`, it is used in `RelayHub`'s `relay` function.

Reply:
I tried to search and it didn't seem possible to do so. The only suggestion the solc maintainer said was to comment it, but we probably can't because it's not us calling the function?
Maybe there is a better way to seem like we use the variable without doing anything?

Reply:
The other option is to accept that this warning will exists and add a comment explaining that those variables aren't necessary in our relayer conditions.

Reply:
Yeah, makes sense